### ISSUE TITLE
Built-in schematics

### ISSUE THREAD

**Issue by Axolotl1107** [cabaletta/baritone#2363]
## Describe your suggestion
Baritone would have built in schematics such as “storage room” or “baritone logo”

## Context
I hate browsing the internet for ages to find a schematic, finally getting one and either discovering it’s a .exe file or that it’s total rubbish. It would be very nice to have a lot of built in baritone compatible schematics such as a storage room, simple house, or the Jesus statue popular on 2b2t.

## Final checklist
- [ ] I know how to properly use check boxes 
- [ ] I have not used any OwO's or UwU's in this issue.

**Comment by CesiumCs** [comment]
I can see why it would be nice to have some example schems to automatically populate the schematics folder

**Comment by calloutplayed** [comment]
Yup it will be a very good thing

**Comment by AaronWebster** [comment]
Might be a good way to document the supported formats as well.

**Comment by scorbett123** [comment]
> Might be a good way to document the supported formats as well.

They are very simple, .schematic.

excluding 1.16 if #2338 is ever merged.

**Comment by l1ving** [comment]
> > Might be a good way to document the supported formats as well.
> 
> They are very simple, .schematic.

No, there are more supported formats than that, documented in the code

**Comment by scorbett123** [comment]
and .schem, forgot that one.

**Comment by AaronWebster** [comment]
Attached is a working example of a minimal portal. If this was to be included as an example in a pull request, where would it go?

[portal-minimal.schematic.gz](https://github.com/cabaletta/baritone/files/5944963/portal-minimal.schematic.gz)

**Comment by l1ving** [comment]
> and .schem, forgot that one.

There were a few other formats in some PR but I'm struggling to find it right now

**Comment by scorbett123** [comment]
> > and .schem, forgot that one.
> 
> There were a few other formats in some PR but I'm struggling to find it right now

There aren't,  just checked.

**Comment by l1ving** [comment]
> There aren't, just checked.

I'm *very* certain someone made a pull adding litematica support or something, but my memory is failing me, I forgot which # it is. I suppose it's not in official Baritone, which matters.

**Comment by scorbett123** [comment]
Yea, hasn't been merged yet though.

**Comment by scorbett123** [comment]
> but my memory is failing me, I forgot which # it is. I suppose it's not in official Baritone, which matters.
Wait, you remember all of the issue / pr numbers

**Comment by l1ving** [comment]
I try to. #971 is burned into my brain because of how many stupid duplicates it has

**Comment by EmersonDove** [comment]
> > There aren't, just checked.
> 
> I'm _very_ certain someone made a pull adding litematica support or something, but my memory is failing me, I forgot which # it is. I suppose it's not in official Baritone, which matters.

Saw the mention, if it's better as a pull request to the master I can fix that.
